aea/cli/test.py

```diff
@@ -116,7 +142,92 @@ def by_path(
     """Executes a test suite of a package specified by a path."""
     click.echo(f"Executing tests of package at {path}'...")
     full_path = Path(ctx.cwd) / Path(path)
-    test_package_by_path(full_path, args, packages_dir=Path(ctx.registry_path))
+    test_package_by_path(
+        full_path,
+        args,
+        packages_dir=Path(ctx.registry_path),
+        cov=ctx.config.get("cov", False),
+    )
+
+
+@test.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    ),
+)
```

Comment:
`@pytest_args` does not work as intended so not we pass pytest args like this, this config allows the command to take in args other then defined here